src/proxy/query.go

```diff
@@ -342,9 +345,16 @@ func queryStat(node sqlparser.Statement, err error) {
 	default:
 		command = "Unsupport"
 	}
+	queryTime := time.Since(timeStart)
 	if err != nil {
+		if queryTime > slowQueryTime {
+			monitor.SlowQueryTotalCounterInc(command, "Error")
+		}
 		monitor.QueryTotalCounterInc(command, "Error")
 	} else {
+		if queryTime > slowQueryTime {
+			monitor.SlowQueryTotalCounterInc(command, "OK")
```

Comment:
You need add test in the query_test.go to the logic which is core in slowlog, and the logic is not coveraged.
it is strange to that the travis-ci is passed